packages/@aws-cdk/aws-cloud9/test/integ.cloud9.ts

```diff
@@ -6,18 +7,36 @@ export class Cloud9Env extends cdk.Stack {
   constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
     super(scope, id, props);
 
-    const vpc = new ec2.Vpc(this, 'VPC', {
-      maxAzs: 2,
-      natGateways: 1,
-    });
+    const vpc = getOrCreateVpc(this);
 
     // create a cloud9 ec2 environment in a new VPC
-    const c9env = new cloud9.Ec2Environment(this, 'C9Env', { vpc });
+    const c9env = new cloud9.Ec2Environment(this, 'C9Env', {
+      vpc,
+      repositories: [
+        {
+          repository: new codecommit.Repository(this, 'Repo', {
+            repositoryName: 'foo',
+          }),
+          path: '/foo',
+        },
+      ],
+    });
     new cdk.CfnOutput(this, 'URL', { value: c9env.ideUrl });
     new cdk.CfnOutput(this, 'ARN', { value: c9env.ec2EnvironmentArn });
   }
 }
 
+function getOrCreateVpc(scope: cdk.Construct): ec2.IVpc {
+  // use an existing vpc or create a new one
+  const vpc = scope.node.tryGetContext('use_default_vpc') === '1' ?
+    ec2.Vpc.fromLookup(scope, 'Vpc', { isDefault: true }) :
+    scope.node.tryGetContext('use_vpc_id') ?
+      ec2.Vpc.fromLookup(scope, 'Vpc', { vpcId: scope.node.tryGetContext('use_vpc_id') }) :
+      new ec2.Vpc(scope, 'Vpc', { maxAzs: 3, natGateways: 1 });
+
+  return vpc;
+}
```

Comment:
Please revert these changes. If you want the VPC to be smaller, maybe pass `maxAzs: 1` when creating it? (And/or other options, like `natGateways: 0`, etc.).